config/jobs/ocp-automation/openshift-install-power/openshift-install-power-postsubmit.yaml

```diff
@@ -4,7 +4,7 @@ postsubmits:
       decorate: true
       run_if_changed: 'docs'
       branches:
-        - ^master$
+        - ^main$
```

Comment:
I see multiple branches in the repo like devel, main etc.. and devel being the default one, so wondering why only the main branch is mentioned here?

btw: this job is failing for years - https://prow.ppc64le-cloud.org/job-history/s3/prow-logs/logs/update-github-pages-submodule, not sure if someone is really maintaining this job

cc @sudeeshjohn

Reply:
Docs are pulled only from main/master.